scripts/check_translations.py

```
@@ -0,0 +1,105 @@
+
+
+def placeholders(value: str) -> Counter[str]:
+    return Counter(match.group(0)[-1].lower() for match in FORMAT.finditer(value.replace("%%", "")))
+
+
+def load(directory: Path) -> tuple[dict[str, ET.Element], list[str]]:
+    resources: dict[str, ET.Element] = {}
+    errors: list[str] = []
+    for file in sorted(directory.glob("*.xml")):
+        try:
+            root = ET.parse(file).getroot()
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)